pkg/cmd/issue/view/presentation_issue.go

```diff
@@ -0,0 +1,122 @@
+package view
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/cli/cli/v2/api"
+	prShared "github.com/cli/cli/v2/pkg/cmd/pr/shared"
+	"github.com/cli/cli/v2/pkg/iostreams"
+)
+
+type PresentationIssue struct {
+	Title          string
+	Number         int
+	CreatedAt      time.Time
+	Comments       api.Comments
```

Comment:
I think it's a bit leaky that we've gone to all this work the separate our `PresentationIssue` from our `api` domain and we're holding on to `api.Comments` in here. I'd need to take some time to investigate a bit more before offering any kind of suggestion though.